### ISSUE TITLE
ScreenReader(NVDA + chrome) does not announce header cell of the table

### ISSUE THREAD

**Issue by srjonnal** [angular-ui/ui-grid#7171]
This is a duplicate of https://github.com/angular-ui/ui-grid/issues/7082. Opening a new issue with details since previous issue is closed. 

While focus is on the header cell of the table, screen reader announces as Column menu. It should be announced as header cell. 

![image](https://user-images.githubusercontent.com/54173649/79122560-9a547400-7db5-11ea-82ac-4de637c3bfd4.png)

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
